dds_web/commands.py

```diff
@@ -1328,7 +1328,7 @@ def monitor_usage():
 @flask.cli.with_appcontext
 def restart_redis_worker():
     """
-    This function restarts the redis worker intialized by the Flask application.
+    This function restarts the redis worker(s) intialized by the Flask application.
```

Comment:
What about this command running in the foreground? 
Or maybe it should be fixed in its own PR.

Reply:
@rv0lt

Reply:
From sprint review: Add clarification that this is intended for development purposes only.